runtime/ddr/blacklist

```diff
@@ -27,6 +27,44 @@ file:<built-in>
 # zlib defines the type 'Byte' which conflicts with java.lang.Byte.
 file:*/zlib/zconf.h
 
-# Ignore types that are troublesome or not required. 
+# Ignore types that are troublesome or not required.
 type:__gnu*
 type:*__va_list*
+type:*vc_attributes*
+type:*<*
+type:tag*
+
+type:*API_WORKING_SET_*
+type:*MINIDUMP_*
+type:*OSVERSIONINFOEXW
+type:*PDH_FMT_COUNTER*
+type:*SAFEARRAY*
+type:*SCOPE_TABLE*
+type:*TIME_ZONE_INFORMATION
+type:*UNWIND_*
+type:*WIN32_FIND_DATAW
+type:pthreadlocinfo
+type:threadlocaleinfostruct
+
+# Ignore one (or all) of sets of types whose names differ only in case
+# (so we don't try to create Java source files whose names differ only
+# in case which isn't possible with case-insensitive file-systems).
+
+type:ADDRINFOW
+type:addrinfoW
+type:BYTE
+type:Byte
+type:byte
+type:classTableEntry
+type:IN6_ADDR
+type:IN_ADDR
+type:SIZE_T
+type:SOCKADDR*
+type:UINT
+type:uInt
+type:ULONG
+type:uLong
+type:UserDataWrapper
+type:userDataWrapper
```

Comment:
Do both versions need to be excluded? We should keep one for debugging purposes.

Reply:
I'll fix this along with classTableEntry in that separate PR.

Reply:
I'll change the  struct tag from `userDataWrapper` to `UserDataWrapper` so we don't have to blacklist either type.